test/nested-pages.html

```diff
@@ -24,10 +24,9 @@ <h1>header</h1>
           type: Number,
           value: 0,
         },
-      },  
+      },
       refire: function(e, detail) {
-        this.items = e.target.items
-        this.fire('iron-items-changed', e)
+        this.items = e.target.items this.fire('iron-items-changed', e)
```

Comment:
This was broken by reformatting because of a missing semicolon.

Reply:
nice catch!